contracts/ownership/Superuser.sol

```diff
@@ -0,0 +1,66 @@
+pragma solidity ^0.4.23;
+
+
+import "./Ownable.sol";
+import "./rbac/RBAC.sol";
+
+
+/**
+ * @title Superuser
+ * @dev The Superuser contract defines a single superuser who can transfer the ownership 
+ * @dev of a contract to a new address, even if he is not the owner. 
+ * @dev A superuser can transfer his role to a new address. 
+ */
+contract Superuser is Ownable, RBAC {
+  event SuperuserTransferred(address indexed previousSuperuser, address indexed newSuperuser);
+  event OwnershipTransferred(address indexed previousOwner, address indexed newOwner);
+
+  string public constant ROLE_SUPERUSER = "superuser";
+
+  constructor () {
+    addRole(msg.sender, ROLE_SUPERUSER);
+  }
+
+  /**
+   * @dev Throws if called by any account that's not a superuser.
+   */
+  modifier onlySuperuser() {
+    checkRole(msg.sender, ROLE_SUPERUSER);
+    _;
+  }
+
+    /**
+   * @dev getter to determine if address has superuser role
+   */
+  function superuser(address addr)
+    public
+    view
+    returns (bool)
+  {
+    return hasRole(addr, ROLE_SUPERUSER);
+  }
+
+  /**
+   * @dev Allows the current superuser to transfer his role to a newSuperuser.
+   * @param newSuperuser The address to transfer ownership to.
+   */
+  function transferSuperuser(address newSuperuser) 
```

Comment:
The convention is to prefix arguments with `_` to keep them visually separate from variables in the function body. make this `_newSuperuser`?

likewise for the various other arguments in this file

Reply:
Perfect! Same here, trying to follow the style of Ownable.sol and Whitelist.sol :)